web interface no longer working

I have been running calibre as part of my media server suite for some time now and sometime in the past week it suddenly stopped responding on the web gui... I've blown away the container and recreated it multiple times to no avail... below are the most recent boot logs from the container, you'll see the RDP connection is working at the end of the log, pinged it just to confirm at least that side still worked.

------------------------------------------------------------------------
*** Running /etc/my_init.d/00_config.sh...

Current default time zone: 'Etc/UTC'
Local time is now:      Tue Jun 20 19:40:38 UTC 2017.
Universal Time is now:  Tue Jun 20 19:40:38 UTC 2017.

*** Running /etc/my_init.d/01_user_config.sh...
usermod: no changes
usermod: no changes
*** Running /etc/my_init.d/02_app_config.sh...
*** Running /etc/my_init.d/firstrun.sh...
EDGE requested, updating to latest version
2017-06-20 19:40:38 URL:https://raw.githubusercontent.com/kovidgoyal/calibre/master/setup/linux-installer.py [27407/27407] -> "-" [1]
Creating symlinks...
	Symlinking /opt/calibre/ebook-meta to /usr/bin/ebook-meta
	Symlinking /opt/calibre/web2disk to /usr/bin/web2disk
	Symlinking /opt/calibre/calibre-parallel to /usr/bin/calibre-parallel
	Symlinking /opt/calibre/ebook-edit to /usr/bin/ebook-edit
	Symlinking /opt/calibre/lrs2lrf to /usr/bin/lrs2lrf
	Symlinking /opt/calibre/calibredb to /usr/bin/calibredb
	Symlinking /opt/calibre/calibre to /usr/bin/calibre
	Symlinking /opt/calibre/ebook-convert to /usr/bin/ebook-convert
	Symlinking /opt/calibre/fetch-ebook-metadata to /usr/bin/fetch-ebook-metadata
	Symlinking /opt/calibre/calibre-smtp to /usr/bin/calibre-smtp
	Symlinking /opt/calibre/calibre-debug to /usr/bin/calibre-debug
	Symlinking /opt/calibre/calibre-server to /usr/bin/calibre-server
	Symlinking /opt/calibre/ebook-device to /usr/bin/ebook-device
	Symlinking /opt/calibre/lrfviewer to /usr/bin/lrfviewer
	Symlinking /opt/calibre/ebook-polish to /usr/bin/ebook-polish
	Symlinking /opt/calibre/ebook-viewer to /usr/bin/ebook-viewer
	Symlinking /opt/calibre/markdown-calibre to /usr/bin/markdown-calibre
	Symlinking /opt/calibre/lrf2lrs to /usr/bin/lrf2lrs
	Symlinking /opt/calibre/calibre-customize to /usr/bin/calibre-customize
Setting up command-line completion...
Failed to find directory to install bash completions, using default.
Installing bash completion to: /usr/share/bash-completion/completions/calibre
Setting up desktop integration...

____________________ WARNING ____________________
Setting up desktop integration failed with error:
__________________________________________________


	Traceback (most recent call last):
	  File "site-packages/calibre/linux.py", line 813, in setup_desktop_integration
	  File "site-packages/calibre/linux.py", line 798, in install_icons
	  File "site-packages/calibre/linux.py", line 792, in install_single_icon
	  File "subprocess.py", line 536, in check_call
	  File "subprocess.py", line 523, in call
	  File "subprocess.py", line 711, in __init__
	  File "subprocess.py", line 1343, in _execute_child
	OSError: [Errno 2] No such file or directory

Creating un-installer: /usr/bin/calibre-uninstall


There were 1 warnings

* Setting up desktop integration failed with error:

Installing to /opt/calibre
Downloading tarball signature securely...
Will download and install calibre-3.0.0-x86_64.txz 
Downloading calibre-3.0.0-x86_64.txz 
Downloaded 5.009763% 
Downloaded 10.019525% 
Downloaded 15.029288% 
Downloaded 20.039050% 
Downloaded 25.048813% 
Downloaded 30.058575% 
Downloaded 35.068338% 
Downloaded 40.078100% 
Downloaded 45.087863% 
Downloaded 50.097625% 
Downloaded 55.107388% 
Downloaded 60.117151% 
Downloaded 65.126913% 
Downloaded 70.136676% 
Downloaded 75.146438% 
Downloaded 80.156201% 
Downloaded 85.165963% 
Downloaded 90.175726% 
Downloaded 95.185488% 
Downloaded 60502668 bytes 
Checking downloaded file integrity... 
Extracting files to /opt/calibre ...
Extracting application files... 
Run "calibre" to start calibre 
*** Running /etc/rc.local...
*** Booting runit daemon...
*** Runit started as PID 70
ok: down: openbox: 0s, normally up, want up
Starting xrdp!
Error: Can't open display: :1
Error: Can't open display: :1
guacd[86]: INFO:	Guacamole proxy daemon (guacd) version 0.9.7 started
guacd[86]: INFO:	Listening on host 127.0.0.1, port 4822
Error: Can't open display: :1
InitOutput:

X11rdp, an X server for xrdp
Version 0.9.0
Copyright (C) 2005-2015 Jay Sorg
See http://www.xrdp.org for information on xrdp.
Underlying X server release 10903000, The X.Org Foundation
Screen width 1920 height 1080 depth 16 bpp 16
dpix 100 dpiy 100
buffer size 4147200
E: [pulseaudio] core-util.c: Failed to connect to system bus: Failed to connect to socket /var/run/dbus/system_bus_socket: No such file or directory
E: [pulseaudio] core-util.c: Failed to connect to system bus: Failed to connect to socket /var/run/dbus/system_bus_socket: No such file or directory
E: [pulseaudio] core-util.c: Failed to connect to system bus: Failed to connect to socket /var/run/dbus/system_bus_socket: No such file or directory
E: [pulseaudio] core-util.c: Failed to connect to system bus: Failed to connect to socket /var/run/dbus/system_bus_socket: No such file or directory
E: [pulseaudio] core-util.c: Failed to connect to system bus: Failed to connect to socket /var/run/dbus/system_bus_socket: No such file or directory
E: [pulseaudio] core-util.c: Failed to connect to system bus: Failed to connect to socket /var/run/dbus/system_bus_socket: No such file or directory
E: [pulseaudio] core-util.c: Failed to connect to system bus: Failed to connect to socket /var/run/dbus/system_bus_socket: No such file or directory
E: [pulseaudio] core-util.c: Failed to connect to system bus: Failed to connect to socket /var/run/dbus/system_bus_socket: No such file or directory
E: [pulseaudio] core-util.c: Failed to connect to system bus: Failed to connect to socket /var/run/dbus/system_bus_socket: No such file or directory
E: [pulseaudio] core-util.c: Failed to connect to system bus: Failed to connect to socket /var/run/dbus/system_bus_socket: No such file or directory
E: [pulseaudio] core-util.c: Failed to connect to system bus: Failed to connect to socket /var/run/dbus/system_bus_socket: No such file or directory
20/06/17 19:41:03 kill disconencted [0] timeout [60] sec
pRRScrPriv 0x89dcf0
rdpScreenInit: ret 1
InitOutput: out
E: sink_process_msg: not running
E: close_send:
[dix] Could not init font path element /usr/share/fonts/X11/misc/, removing from list!
[dix] Could not init font path element /usr/share/fonts/X11/TTF/, removing from list!
[dix] Could not init font path element /usr/share/fonts/X11/OTF/, removing from list!
W: [pulseaudio] authkey.c: Failed to open cookie file '/nobody/.config/pulse/cookie': No such file or directory
W: [pulseaudio] authkey.c: Failed to load authorization key '/nobody/.config/pulse/cookie': No such file or directory
W: [pulseaudio] authkey.c: Failed to open cookie file '/nobody/.pulse-cookie': No such file or directory
W: [pulseaudio] authkey.c: Failed to load authorization key '/nobody/.pulse-cookie': No such file or directory
[dix] Could not init font path element /usr/share/fonts/X11/100dpi/, removing from list!
[dix] Could not init font path element /usr/share/fonts/X11/75dpi/, removing from list!
W: [pulseaudio] server-lookup.c: Unable to contact D-Bus: org.freedesktop.DBus.Error.Spawn.ExecFailed: //bin/dbus-launch terminated abnormally without any error message
rdpSpriteDeviceCursorInitialize:
W: [pulseaudio] main.c: Unable to contact D-Bus: org.freedesktop.DBus.Error.Spawn.ExecFailed: //bin/dbus-launch terminated abnormally without any error message
Jun 20 19:41:03 calibre syslog-ng[90]: syslog-ng starting up; version='3.5.3'
rdpPointerWarpCursor:
rdpPointerNewEventScreen:
InitInput:
rdpMouseCtrl:
rdpChangeKeyboardControl:
Obt-Message: Xinerama extension is not present on the server
Fontconfig warning: ignoring C.UTF-8: not a valid language tag
Openbox-Message: Unable to find a valid menu file "/var/lib/openbox/debian-menu.xml"
Starting X app...
rdpChangeKeyboardControl:
Usage: calibre-server [options] [path to library folder...]

Start the calibre Content server. The calibre Content server exposes your
calibre libraries over the internet. You can specify the path to the library
folders as arguments to calibre-server. If you do not specify any paths, all the
libraries that the main calibre program knows about will be used.


Whenever you pass arguments to calibre-server that have spaces in them, enclose the arguments in quotation marks. For example: "/some path/with spaces"

calibre-server: error: no such option: --with-library
Jun 20, 2017 7:41:03 PM org.apache.catalina.startup.ClassLoaderFactory validateFile
WARNING: Problem with directory [/usr/share/tomcat7/common/classes], exists: [false], isDirectory: [false], canRead: [false]
Jun 20, 2017 7:41:03 PM org.apache.catalina.startup.ClassLoaderFactory validateFile
WARNING: Problem with directory [/usr/share/tomcat7/common], exists: [false], isDirectory: [false], canRead: [false]
rdpRRGetInfo:
rdpRRRegisterSize: width 1920 height 1080
rdpRRGetPanning:
Fontconfig warning: ignoring C.UTF-8: not a valid language tag
Jun 20, 2017 7:41:03 PM org.apache.catalina.startup.ClassLoaderFactory validateFile
WARNING: Problem with directory [/usr/share/tomcat7/server/classes], exists: [false], isDirectory: [false], canRead: [false]
Jun 20, 2017 7:41:03 PM org.apache.catalina.startup.ClassLoaderFactory validateFile
WARNING: Problem with directory [/usr/share/tomcat7/server], exists: [false], isDirectory: [false], canRead: [false]
Jun 20, 2017 7:41:03 PM org.apache.catalina.startup.ClassLoaderFactory validateFile
WARNING: Problem with directory [/usr/share/tomcat7/shared/classes], exists: [false], isDirectory: [false], canRead: [false]
Jun 20, 2017 7:41:03 PM org.apache.catalina.startup.ClassLoaderFactory validateFile
WARNING: Problem with directory [/usr/share/tomcat7/shared], exists: [false], isDirectory: [false], canRead: [false]
Jun 20, 2017 7:41:03 PM org.apache.catalina.startup.Catalina initDirs
SEVERE: Cannot find specified temporary folder at /tmp/tomcat7-tomcat7-tmp
ok: run: openbox: (pid 89) 1s
rdpRRGetPanning:
Jun 20, 2017 7:41:04 PM org.apache.coyote.AbstractProtocol init
INFO: Initializing ProtocolHandler ["http-bio-8080"]
Jun 20, 2017 7:41:04 PM org.apache.catalina.startup.Catalina load
INFO: Initialization processed in 612 ms
Jun 20, 2017 7:41:04 PM org.apache.catalina.core.StandardService startInternal
INFO: Starting service Catalina
Jun 20, 2017 7:41:04 PM org.apache.catalina.core.StandardEngine startInternal
INFO: Starting Servlet Engine: Apache Tomcat/7.0.52 (Ubuntu)
Jun 20, 2017 7:41:04 PM org.apache.catalina.startup.HostConfig deployWAR
INFO: Deploying web application archive /var/lib/tomcat7/webapps/ROOT.war
Jun 20, 2017 7:41:04 PM org.apache.catalina.core.StandardContext postWorkDirectory
WARNING: Failed to create work directory [/var/lib/tomcat7/work/Catalina/localhost/_] for context []
Warning: Missing charsets in String to FontSet conversion
19:41:07.298 [localhost-startStop-1] INFO  o.g.g.n.b.r.a.BasicTokenSessionMap - Sessions will expire after 60 minutes of inactivity.
19:41:07.432 [localhost-startStop-1] WARN  o.g.g.n.b.extension.ExtensionModule - The "auth-provider" and "lib-directory" properties are now deprecated. Please use the "extensions" and "lib" directories within GUACAMOLE_HOME instead.
19:41:07.598 [localhost-startStop-1] INFO  o.g.g.n.b.w.t.WebSocketTunnelModule - Loading Tomcat 7 WebSocket support...
Jun 20, 2017 7:41:07 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory register
INFO: Registering org.glyptodon.guacamole.net.basic.rest.activeconnection.ActiveConnectionRESTService as a root resource class
Jun 20, 2017 7:41:07 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory register
INFO: Registering org.glyptodon.guacamole.net.basic.rest.clipboard.ClipboardRESTService as a root resource class
Jun 20, 2017 7:41:07 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory register
INFO: Registering org.glyptodon.guacamole.net.basic.rest.connectiongroup.ConnectionGroupRESTService as a root resource class
Jun 20, 2017 7:41:07 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory register
INFO: Registering org.glyptodon.guacamole.net.basic.rest.connection.ConnectionRESTService as a root resource class
Jun 20, 2017 7:41:07 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory register
INFO: Registering org.glyptodon.guacamole.net.basic.rest.language.LanguageRESTService as a root resource class
Jun 20, 2017 7:41:07 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory register
INFO: Registering org.glyptodon.guacamole.net.basic.rest.schema.SchemaRESTService as a root resource class
Jun 20, 2017 7:41:07 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory register
INFO: Registering org.glyptodon.guacamole.net.basic.rest.auth.TokenRESTService as a root resource class
Jun 20, 2017 7:41:07 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory register
INFO: Registering org.glyptodon.guacamole.net.basic.rest.user.UserRESTService as a root resource class
Jun 20, 2017 7:41:07 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory register
INFO: Registering org.codehaus.jackson.jaxrs.JacksonJsonProvider as a provider class
Jun 20, 2017 7:41:07 PM com.sun.jersey.server.impl.application.WebApplicationImpl _initiate
INFO: Initiating Jersey application, version 'Jersey: 1.17.1 02/28/2013 12:47 PM'
Jun 20, 2017 7:41:08 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory getComponentProvider
INFO: Binding org.codehaus.jackson.jaxrs.JacksonJsonProvider to GuiceManagedComponentProvider with the scope "Singleton"
Jun 20, 2017 7:41:08 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory getComponentProvider
INFO: Binding org.glyptodon.guacamole.net.basic.rest.activeconnection.ActiveConnectionRESTService to GuiceManagedComponentProvider with the scope "PerRequest"
Jun 20, 2017 7:41:08 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory getComponentProvider
INFO: Binding org.glyptodon.guacamole.net.basic.rest.clipboard.ClipboardRESTService to GuiceManagedComponentProvider with the scope "PerRequest"
Jun 20, 2017 7:41:08 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory getComponentProvider
INFO: Binding org.glyptodon.guacamole.net.basic.rest.connectiongroup.ConnectionGroupRESTService to GuiceManagedComponentProvider with the scope "PerRequest"
Jun 20, 2017 7:41:08 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory getComponentProvider
INFO: Binding org.glyptodon.guacamole.net.basic.rest.connection.ConnectionRESTService to GuiceManagedComponentProvider with the scope "PerRequest"
Jun 20, 2017 7:41:08 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory getComponentProvider
INFO: Binding org.glyptodon.guacamole.net.basic.rest.language.LanguageRESTService to GuiceManagedComponentProvider with the scope "PerRequest"
Jun 20, 2017 7:41:08 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory getComponentProvider
INFO: Binding org.glyptodon.guacamole.net.basic.rest.schema.SchemaRESTService to GuiceManagedComponentProvider with the scope "PerRequest"
Jun 20, 2017 7:41:08 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory getComponentProvider
INFO: Binding org.glyptodon.guacamole.net.basic.rest.auth.TokenRESTService to GuiceManagedComponentProvider with the scope "PerRequest"
Jun 20, 2017 7:41:08 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory getComponentProvider
INFO: Binding org.glyptodon.guacamole.net.basic.rest.user.UserRESTService to GuiceManagedComponentProvider with the scope "PerRequest"
Jun 20, 2017 7:41:08 PM org.apache.jasper.EmbeddedServletOptions <init>
SEVERE: The scratchDir you specified: /var/lib/tomcat7/work/Catalina/localhost/_ is unusable.
Jun 20, 2017 7:41:08 PM org.apache.catalina.startup.HostConfig deployWAR
INFO: Deploying web application archive /var/lib/tomcat7/webapps/guacamole.war
Jun 20, 2017 7:41:09 PM org.apache.catalina.core.StandardContext postWorkDirectory
WARNING: Failed to create work directory [/var/lib/tomcat7/work/Catalina/localhost/guacamole] for context [/guacamole]
DBusExport: Failed to connect to DBUS session bus, with error: org.freedesktop.DBus.Error.Spawn.ExecFailed: /usr/bin/dbus-launch terminated abnormally without any error message
19:41:10.441 [localhost-startStop-1] INFO  o.g.g.n.b.r.a.BasicTokenSessionMap - Sessions will expire after 60 minutes of inactivity.
19:41:10.555 [localhost-startStop-1] WARN  o.g.g.n.b.extension.ExtensionModule - The "auth-provider" and "lib-directory" properties are now deprecated. Please use the "extensions" and "lib" directories within GUACAMOLE_HOME instead.
19:41:10.584 [localhost-startStop-1] INFO  o.g.g.n.b.w.t.WebSocketTunnelModule - Loading Tomcat 7 WebSocket support...
Jun 20, 2017 7:41:10 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory register
INFO: Registering org.glyptodon.guacamole.net.basic.rest.activeconnection.ActiveConnectionRESTService as a root resource class
Jun 20, 2017 7:41:10 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory register
INFO: Registering org.glyptodon.guacamole.net.basic.rest.clipboard.ClipboardRESTService as a root resource class
Jun 20, 2017 7:41:10 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory register
INFO: Registering org.glyptodon.guacamole.net.basic.rest.connectiongroup.ConnectionGroupRESTService as a root resource class
Jun 20, 2017 7:41:10 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory register
INFO: Registering org.glyptodon.guacamole.net.basic.rest.connection.ConnectionRESTService as a root resource class
Jun 20, 2017 7:41:10 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory register
INFO: Registering org.glyptodon.guacamole.net.basic.rest.language.LanguageRESTService as a root resource class
Jun 20, 2017 7:41:10 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory register
INFO: Registering org.glyptodon.guacamole.net.basic.rest.schema.SchemaRESTService as a root resource class
Jun 20, 2017 7:41:10 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory register
INFO: Registering org.glyptodon.guacamole.net.basic.rest.auth.TokenRESTService as a root resource class
Jun 20, 2017 7:41:10 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory register
INFO: Registering org.glyptodon.guacamole.net.basic.rest.user.UserRESTService as a root resource class
Jun 20, 2017 7:41:10 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory register
INFO: Registering org.codehaus.jackson.jaxrs.JacksonJsonProvider as a provider class
Jun 20, 2017 7:41:10 PM com.sun.jersey.server.impl.application.WebApplicationImpl _initiate
INFO: Initiating Jersey application, version 'Jersey: 1.17.1 02/28/2013 12:47 PM'
Jun 20, 2017 7:41:11 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory getComponentProvider
INFO: Binding org.codehaus.jackson.jaxrs.JacksonJsonProvider to GuiceManagedComponentProvider with the scope "Singleton"
Jun 20, 2017 7:41:11 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory getComponentProvider
INFO: Binding org.glyptodon.guacamole.net.basic.rest.activeconnection.ActiveConnectionRESTService to GuiceManagedComponentProvider with the scope "PerRequest"
Jun 20, 2017 7:41:11 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory getComponentProvider
INFO: Binding org.glyptodon.guacamole.net.basic.rest.clipboard.ClipboardRESTService to GuiceManagedComponentProvider with the scope "PerRequest"
Jun 20, 2017 7:41:11 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory getComponentProvider
INFO: Binding org.glyptodon.guacamole.net.basic.rest.connectiongroup.ConnectionGroupRESTService to GuiceManagedComponentProvider with the scope "PerRequest"
Jun 20, 2017 7:41:11 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory getComponentProvider
INFO: Binding org.glyptodon.guacamole.net.basic.rest.connection.ConnectionRESTService to GuiceManagedComponentProvider with the scope "PerRequest"
Jun 20, 2017 7:41:11 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory getComponentProvider
INFO: Binding org.glyptodon.guacamole.net.basic.rest.language.LanguageRESTService to GuiceManagedComponentProvider with the scope "PerRequest"
Jun 20, 2017 7:41:11 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory getComponentProvider
INFO: Binding org.glyptodon.guacamole.net.basic.rest.schema.SchemaRESTService to GuiceManagedComponentProvider with the scope "PerRequest"
Jun 20, 2017 7:41:11 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory getComponentProvider
INFO: Binding org.glyptodon.guacamole.net.basic.rest.auth.TokenRESTService to GuiceManagedComponentProvider with the scope "PerRequest"
Jun 20, 2017 7:41:11 PM com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory getComponentProvider
INFO: Binding org.glyptodon.guacamole.net.basic.rest.user.UserRESTService to GuiceManagedComponentProvider with the scope "PerRequest"
Jun 20, 2017 7:41:11 PM org.apache.jasper.EmbeddedServletOptions <init>
SEVERE: The scratchDir you specified: /var/lib/tomcat7/work/Catalina/localhost/guacamole is unusable.
Jun 20, 2017 7:41:11 PM org.apache.coyote.AbstractProtocol start
INFO: Starting ProtocolHandler ["http-bio-8080"]
Jun 20, 2017 7:41:11 PM org.apache.catalina.startup.Catalina start
INFO: Server startup in 6908 ms
19:41:29.344 [http-bio-8080-exec-9] INFO  o.g.g.n.b.rest.auth.TokenRESTService - User "b415eff4-ca94-4f07-bab0-3be8677316b6" successfully authenticated from 192.168.1.110.
guacd[228]: INFO:	Protocol "rdp" selected
Jun 20 19:41:29 calibre guacd[228]: Protocol "rdp" selected
guacd[228]: INFO:	Connection ID is "$3431c89a-abc7-42e0-b2f8-98b22cf393a0"
Jun 20 19:41:29 calibre guacd[228]: Connection ID is "$3431c89a-abc7-42e0-b2f8-98b22cf393a0"
guacd[228]: INFO:	No security mode specified. Defaulting to RDP.
Jun 20 19:41:29 calibre guacd[228]: No security mode specified. Defaulting to RDP.
guacd[228]: INFO:	Loading keymap "base"
guacd[228]: INFO:	Loading keymap "en-us-qwerty"
Jun 20 19:41:29 calibre guacd[228]: Loading keymap "base"
Jun 20 19:41:29 calibre guacd[228]: Loading keymap "en-us-qwerty"
Jun 20 19:41:29 calibre XRDP[85]: (85)(140594277594944)[INFO ] A connection received from: 127.0.0.1 port 55186
Jun 20 19:41:29 calibre XRDP[85]: (85)(140594277594944)[INFO ] An established connection closed to endpoint: 127.0.0.1:55186 - socket: 8
Jun 20 19:41:29 calibre XRDP[229]: (229)(140594277594944)[INFO ] An established connection closed to endpoint: NULL:NULL - socket: 7
guacd[228]: INFO:	Starting client
Jun 20 19:41:29 calibre guacd[228]: Starting client
guacd[228]: INFO:	guacsnd connected.
guacd[228]: INFO:	guacdr connected.
Jun 20 19:41:29 calibre guacd[228]: guacsnd connected.
Jun 20 19:41:29 calibre guacd[228]: guacdr connected.
19:41:29.656 [http-bio-8080-exec-1] INFO  o.g.g.net.basic.TunnelRequestService - User "b415eff4-ca94-4f07-bab0-3be8677316b6" successfully connected to "Calibre".
Jun 20 19:41:29 calibre XRDP[229]: (229)(140594277594944)[WARN ] local keymap file for 0x0409 found and dosen't match built in keymap, using local keymap file
guacd[228]: INFO:	Client disconnected
Jun 20 19:41:39 calibre guacd[228]: Client disconnected
Jun 20 19:41:39 calibre XRDP[229]: (229)(140594277594944)[INFO ] An established connection closed to endpoint: 127.0.0.1:55186 - socket: 8
connected to 127.0.0.1:3389
unknown capability type 6
incorrect offset, type:0x06 actual:4 expected:5
Jun 20 19:41:39 calibre XRDP[229]: (229)(140594277594944)[ERROR] Listening socket is in wrong state we terminate listener
logging configuration:
	LogFile:       /var/log/xrdp-ng.log
	LogLevel:      4
	EnableSyslog:  1
	SyslogLevel:   4
[20170620-19:41:29] [INFO ] A connection received from: 127.0.0.1 port 55186
[20170620-19:41:29] [INFO ] An established connection closed to endpoint: NULL:NULL - socket: 7
pub_mod bytes 64
pri_exp 64
colorDepth 0xca01 (0xca00 4bpp 0xca01 8bpp)
postBeta2ColorDepth 0xca01 (0xca00 4bpp 0xca01 8bpp 0xca02 15bpp 0xca03 16bpp 0xca04 24bpp)
highColorDepth 0x0010 (0x0004 4bpp 0x0008 8bpp 0x000f 15bpp 0x0010 16 bpp 0x0018 24bpp)
supportedColorDepths 0x0007 (0x0001 24bpp 0x0002 16bpp 0x0004 15bpp 0x0008 32bpp)
earlyCapabilityFlags 0x0001 (0x0002 want32)
got client client connection type 0x00000000
xrdp_sec_process_mcs_data_CS_SECURITY:
  client supports 40 bit encryption
  client supports 128 bit encryption
  client supports fips encryption
  client and server support low crypt, using low crypt
xrdp_sec_out_mcs_data: using 512 bit RSA key
xrdp_load_keyboard_layout:
xrdp_load_keyboard_layout: model [] variant [] layout [us]
xrdp_sec_establish_keys:
xrdp_caps_process_pointer: client supports new(color) cursor
xrdp_process_offscreen_bmpcache: support level 1 cache size 7864320 MB cache entries 100
unknown in xrdp_rdp_process_data 43
yeah, up_and_running
[20170620-19:41:29] [DEBUG] xrdp_000000e5_wm_login_mode_event_00000001
xrdp:xrdp_mm [3339070641]: xrdp_mm_create: bpp 16 mcs_connection_type 0 jpeg_codec_id 0 v3_codec_id 0 rfx_codec_id 0 h264_codec_id 0
[20170620-19:41:29] [WARN ] local keymap file for 0x0409 found and dosen't match built in keymap, using local keymap file
xrdp_wm_login_mode_changed: login_mode is 0
in xrdp_wm_init: 
   xrdp_wm_init: no autologin / auto run detected, draw login window
xrdp:xrdp_cache [3339070652]: xrdp_cache_add_bitmap: reset detected cache_id 2
xrdp:xrdp_cache [3339070653]: xrdp_cache_add_bitmap: reset detected cache_id 1
out xrdp_wm_init: 
xrdp_wm_login_mode_changed: login_mode is 1
[20170620-19:41:39] [INFO ] An established connection closed to endpoint: 127.0.0.1:55186 - socket: 8
xrdp_mcs_disconnect - socket closed
[20170620-19:41:39] [DEBUG] xrdp_mm_module_cleanup
xrdp:xrdp_encoder [3339080498]: xrdp_encoder_delete:
[20170620-19:41:39] [ERROR] Listening socket is in wrong state we terminate listener

--- a/README.md
+++ b/README.md
@@ -12,7 +12,7 @@ On unRaid, install from the Community Repositories and enter the app folder loca
 On other platforms, you can run this docker with the following command:
 
 ```
-docker run -d --name="RDP-Calibre" -e EDGE="0" -e WIDTH="1280" -e HEIGHT="720" -v /path/to/config:/config:rw -v /etc/localtime:/etc/localtime:ro -p XXXX:8080 -p YYYY:8081 aptalca/docker-rdp-calibre
+docker run -d --name="RDP-Calibre" -e EDGE="0" -e WIDTH="1280" -e HEIGHT="720" -v /path/to/config:/config:rw -e TZ=America/New_York -p XXXX:8080 -p YYYY:8081 aptalca/docker-rdp-calibre
 ```
 
 ### Setup Instructions
@@ -22,6 +22,7 @@ docker run -d --name="RDP-Calibre" -e EDGE="0" -e WIDTH="1280" -e HEIGHT="720" -
 - If you would like to have the latest updates, change the EDGE variable to "1", and the container will update calibre to the latest version every time it is restarted
 - If you'd like to change the resolution for the GUI, you can modify the WIDTH and HEIGHT variables
 - IMPORTANT: On first start, select "/config" as the library location in the Calibre wizard
+- Calibre webserver can be enabled from the calibre gui under Preferences/Sharing over the net. **Port has to be set to 8081**
 
 You can access the GUI by pointing your web browser to http://SERVERIP:XXXX/#/client/c/Calibre
 
@@ -30,16 +31,17 @@ You can access the Calibre webserver by pointing your web browser to http://SERV
 (Replace SERVERIP, XXXX and YYYY with your values)
 
 ### Advanced Features (only for docker pros)
-#### Custom library location:
+#### (Deprecated)Custom library location:
 If you would like to change the library location you need to do a couple of things:
 - First add a new mount point for the library location in the docker run command. Example: -v /path/to/library:/library:rw
 - Then add an environment variable (LIBRARYINTERNALPATH) to specify the internal library location for the webserver. Example: -e LIBRARYINTERNALPATH="/library"
 - When you fire up calibre the first time select your library location. Example: /library  
 
-#### Url Prefix for reverse proxy:
+#### (Deprecated)Url Prefix for reverse proxy:
 - Add an environment variable (URLPREFIX) to docker run to specify the url prefix for the webserver. Example: -e URLPREFIX="/calibre"
 - To access the webserver, go to http://SERVERIP:YYYY/calibre
 
 ### Changelog:
-- 2016-09-16 - Remove X1-lock if exists, which prevents start up following an unclean shutdown
-- 2016-09-15 - Updated base to x11rdp1.3, which now supports clipboard through the left hand side menu (ctrl+alt+shift) - calibre updated to 2.67
++ **2017-06-24:** LIBRARYINTERNALPATH and URLPREFIX variables are deprecated. They were required for the calibre webserver. The new webserver should be enabled from the calibre gui under Preferences/Sharing over the net. **Port has to be set to 8081**
++ **2016-09-16:** Remove X1-lock if exists, which prevents start up following an unclean shutdown
++ **2016-09-15:** Updated base to x11rdp1.3, which now supports clipboard through the left hand side menu (ctrl+alt+shift) - calibre updated to 2.67

--- a/firstrun.sh
+++ b/firstrun.sh
@@ -12,9 +12,3 @@ else
   echo "EDGE requested, updating to latest version"
   wget -nv -O- https://raw.githubusercontent.com/kovidgoyal/calibre/master/setup/linux-installer.py | sudo python -c "import sys; main=lambda:sys.stderr.write('Download failed\n'); exec(sys.stdin.read()); main()"
 fi
-
-if [ -z "$LIBRARYINTERNALPATH" ]; then
-  LIBRARYINTERNALPATH=/config
-fi
-
-/sbin/setuser nobody calibre-server --with-library=$LIBRARYINTERNALPATH --port 8081 --url-prefix=$URLPREFIX &

